File: ClientApp/src/app/report/entity/countbypostatus.ts
export class CountByPostatus {

  public id !: number;
  public postatus !: string;
  public count !: number;
  public perecentage !: number;

  constructor(id:number,postatus:string,count:number,perecentage:number) {
    this.id=id;
    this.postatus=postatus;
    this.count=count;
    this.perecentage=perecentage;
  }

}

File: ServerApp/src/main/java/lk/cooperative/cooperativejaela/report/entity/CountByPostatus.java
package lk.cooperative.cooperativejaela.report.entity;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class CountByPostatus {

    private Integer id;
    private String postatus;
    private Long count;
    private double percentage;

    public CountByPostatus() {  }

    public CountByPostatus(String postatus, Long count) {
        this.postatus = postatus;
        this.count = count;
    }

    public String getPostatus() {
        return postatus;
    }
    public void setPostatus(String postatus) {
        this.postatus = postatus;
    }
    public Long getCount() {
        return count;
    }
    public void setCount(Long count) {
        this.count = count;
    }
    public double getPercentage() {
        return percentage;
    }
    public void setPercentage(double percentage) {
        this.percentage = percentage;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    @Id
    public Integer getId() {
        return id;
    }

}
